CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 10.19
+* Bump content-api-model to 10.19 (content-atom-model 2.4.27)
```

Comment:
Sorry, I was just following the existing conventions of the file. Give me one sec.